### DISCUSSION TITLE
Services option through CLI

### DISCUSSION THREAD

**Discussion by Bajibabup** [webdriverio/codemod#91] [General]
Hi, I have a primary requirement to parametrize the services option, i,e how can we parametrize the concerned service during run time through for npm run command? Example, I have pasted the code below, if we need to choose one of the services (chrome driver vs Sauce) during run time through CLI command, then how can we do that? is it possible to do for wdio?
I referred the documentation: https://webdriver.io/docs/testrunner/#wdio-config. Here also I don't find any option to provide services(like --service Sauce)
Can you help me on it. If any customized code needs to develop then appreciate the samples.
services: [
         ['sauce', {
             sauceConnect: true,
             }],
        'chromedriver',
        'selenium-standalone',
    ],

**Comment by christian-bromann** [comment]
I would recommend to create a config file for testing locally and on Sauce Labs. Another option is to work with environment variables or custom CLI parameter you can then read via `process.argv`.

**Comment by Bajibabup** [comment]
Thanks l!Let me try Env variables or Custom CLI options.Sent from my iPhoneOn Feb 27, 2023, at 6:13 PM, Christian Bromann ***@***.***> wrote:﻿
I would recommend to create a config file for testing locally and on Sauce Labs. Another option is to work with environment variables or custom CLI parameter you can then read via process.argv.
